detox/android/detox/build.gradle

```diff
@@ -131,16 +135,16 @@ if (rootProject.hasProperty('isOfficialDetoxLib') ||
             junitPlatform {
                 filters {
                     engines {
-                        include 'spek2'
+                        include 'spek2', 'junit-vintage'
```

Comment:
Apparently, non-Spek tests were not executed via `gradle`!
https://github.com/spekframework/spek/issues/232#issuecomment-610732158